cmd/capi-checker/main.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"sigs.k8s.io/cluster-api-provider-docker/cmd/capi-checker/pkg/check"
+	"sigs.k8s.io/cluster-api-provider-docker/cmd/versioninfo"
+)
+
+// Constants
+const (
+	KUBECONFIG = "KUBECONFIG"
```

Comment:
the convention in go is to use use either an exported variable like `Kubeconfig` or a local-to-package variable like `kubeconfig`. Depending on what you want (probably `kubeconfig`) please use one of the two instead.